### PR TITLE
Feature/storybook poc/#730629677501460

### PR DESCRIPTION
**Pull request by cryptoB0T**
preview this:
`npm run storybook`

### CODE REVIEW

File: stories/index.stories.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import { bodySectionProps } from './bodySectionProps'
+
+
+storiesOf('Coin', module).add('summary', () => <CoinSummary  />);
+storiesOf('Coin', module).add('detail', () => <CoinDetail  />);
+
+/*
+storiesOf('News', module).add('body section', () => <BodySection bucket={bodySectionProps}  />);
+storiesOf('News', module).add('news body', () => <NewsBody {bodySectionProps.bodyProps}  />);
+storiesOf('News', module).add('news coin tags', () => <NewsBody {bodySectionProps.bodyProps}  />);
```
Comment (cryptoB0T) [inline stories/index.stories.js]: @hanchang I think we will have a hard time reusing some of these components. i have imported them into here so you can play around see how it works.

Comment (hanchang) [reply stories/index.stories.js]: No worries, I think we knew going in that a majority of components would be tricky to reuse, thanks for the heads up!

File: README.md [hunk 2]
```diff
@@ -108,6 +108,19 @@ Note that the roadmap outlined above is subject to change based on user feedback
...
+## Storybook
+
+run storybook in development
+```
+npm run storybook
+```
+
+build static storybook site
+```
+npm run build-storybook -- -o out7650
```
Comment (hanchang) [inline README.md]: What's the `out7650` mean?

**Comment by cryptoB0T** [comment]
Output folder name

File: README.md (hunk 2 above)
Comment (cryptoB0T) [reply README.md]: this is the name of the folder that will be generated. it contains the static site for storybook

**Pull request merged**